behaviours/common.go

```diff
@@ -0,0 +1,72 @@
+package behaviours
+
+// Hookable must be implemented by those entities that attend to all DB Hooks
```

Comment:
why do you need the hookable interface? The thing about events is that you should be able to implement just the ones you need. This interface forces you to implement all of them